config/locales/en.yml

```diff
@@ -3725,6 +3723,33 @@ See the %{link} to find out more about %{sitename}'s features and to start using
         There was a problem setting up your card in our payments gateway.
         Please refresh the page and try again, if it fails a second time,
         please contact us for support.
+    trix:
+      attachFiles: "Attach Files"
+      bold: "Bold"
+      bullets: "Bullets"
+      byte: "Byte"
+      bytes: "Bytes"
+      captionPlaceholder: "Add a caption…"
+      code: "Code"
+      heading1: "Heading"
+      indent: "Increase Level"
+      italic: "Italic"
+      link: "Link"
+      numbers: "Numbers"
+      outdent: "Decrease Level"
+      quote: "Quote"
+      redo: "Redo"
+      remove: "Remove"
+      strike: "Strikethrough"
+      undo: "Undo"
+      unlink: "Unlink"
+      url: "URL"
+      urlPlaceholder: "Enter a URL…"
+      GB: "GB"
+      KB: "KB"
+      MB: "MB"
+      PB: "PB"
+      TB: "TB"
```

Comment:
Thanks for adding Trix translation keys. 

These two old labels seem equivalent to new labels (I haven't checked though)
* "Insert / edit link" -> "Link"
* "Please enter a URL to insert" -> "Enter a URL…"

If so, I think it would help to use the old labels. That way, it will probably be easy for translators to copy the existing labels when adding translations for Trix. Does that make sense?

Reply:
Good idea, that's updated in https://github.com/openfoodfoundation/openfoodnetwork/pull/12734/commits/2d6ffc0ca16459f0d4ee3abdaf82189ddd0a8656 to use the old "Please enter a URL to insert" label.

The "Link" translation is used in two places at once so the "Insert / edit link" label is a bit more awkward, see...

![link-translation](https://github.com/user-attachments/assets/4e9d22f2-81ee-4016-88a7-71c8ec7fcbcc)